packages/editor/src/components/post-text-editor/index.js

```diff
@@ -71,7 +71,7 @@ export class PostTextEditor extends Component {
 		return (
 			<Fragment>
 				<label htmlFor={ `post-content-${ instanceId }` } className="screen-reader-text">
-					{ decodedPlaceholder || __( 'Write your story' ) }
+					{ decodedPlaceholder || __( 'Type text' ) }
```

Comment:
Let's remove the `decodedPlaceholder` from this component and ship.

Reply:
Sounds good; if that means this is otherwise good to go mind explicitly reviewing it now? 😄